src/Orc.Controls/Helpers/DateTimeFormatHelper.cs

```diff
@@ -64,7 +64,7 @@ public static string[] Split(string format, char[] formatCharacters)
             return parts.ToArray();
         }
 
-        public static DateTimeFormatInfo GetDateTimeFormatInfo(string format, bool isDateOnly = false)
+        public static DateTimeFormatInfo GetDateTimeFormatInfo(string format, bool isDateRequired, bool isDateOnly = false)
```

Comment:
Seems like a breaking change. Let's discuss the proper way forward to introduce this.

Reply:
These changes were unnecessary. I canceled them.